module/zfs/metaslab.c

```diff
@@ -146,7 +146,7 @@ static uint_t zfs_mg_fragmentation_threshold = 95;
  * active metaslab that exceeds this threshold will no longer keep its active
  * status allowing better metaslabs to be selected.
  */
-static uint_t zfs_metaslab_fragmentation_threshold = 70;
+static uint_t zfs_metaslab_fragmentation_threshold = 85;
```

Comment:
I suppose with the new curve here should be 77, or you have some other motivation?